modules/ROOT/pages/db/db-configure-connection.adoc

```diff
@@ -3,20 +3,22 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The Database connector can connect to any database for which a JDBC driver is available. To make it easier to connect to the most popular databases, the connector includes a series of different connection types.
+The Database connector can connect to any database for which a JDBC driver is available.
+To make it easier to connect to the most popular databases, the connector includes a series of different connection types.
 
-Examples in this section show configurations for the connection types. They also illustrate more advanced use cases, such as connecting to a global data source, providing a configuration for a JDBC driver, and setting up connection pooling.
+Examples in this section show configurations for common connection types such as MySQL, Oracle, and Microsoft SQL Server.
```

Comment:
"Examples in this section" > "Some of the following examples" or "Some of these examples" or "Some of the examples here"
Avoid referring to "this document" or "this section" or "this topic" or any similar.

Reply:
Accepted "some of the following examples."